runhouse/utils.py

```diff
@@ -652,3 +652,15 @@ def get_gpu_usage(collected_gpus_info: dict, servlet_type: ServletType):
         gpu_usage["utilization_percent"] = gpu_utilization_percent
 
     return gpu_usage
+
+
+class StatusColors(str, Enum):
+    RUNNING = "[green]Running[/green]"
+    SERVER_DOWN = "[orange1]Runhouse server down[/orange1]"
+    TERMINATED = "[red]Terminated[/red]"
+    UNKNOWN = "Unknown"
+    LOCAL_CLUSTER = "[bright_yellow]Local cluster[/bright_yellow]"
+
+
+def get_status_color(status: str):
```

Comment:
could make it a class method? 

```
    @classmethod
    def get_status_color(cls, status: str):
        return getattr(cls, status.upper()).value
```

Reply:
Yeah, will make the change in  cluster status list [PR](https://github.com/run-house/runhouse/pull/1233), because in this PR we move all core logic of clusters list to be static / class methods of `rh.Cluster`.

Reply:
oh didnt mean to move it into cluster, jsut making it a class method of `StatusColors`, but yea doesn't really matter

Reply:
oh I see, fair point, change it